src/equations/shallow_water_wet_dry_1d.jl

```diff
@@ -0,0 +1,677 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    ShallowWaterEquationsWetDry1D(; gravity, H0 = 0, threshold_limiter = nothing threshold_wet = nothing)
+
+Shallow water equations (SWE) in one space dimension. The equations are given by
+```math
+\begin{aligned}
+  \frac{\partial h}{\partial t} + \frac{\partial}{\partial x}(h v) &= 0 \\
+    \frac{\partial}{\partial t}(h v) + \frac{\partial}{\partial x}\left(h v^2 + \frac{g}{2}h^2\right)
+    + g h \frac{\partial b}{\partial x} &= 0
+\end{aligned}
+```
+The unknown quantities of the SWE are the water height ``h`` and the velocity ``v``.
+The gravitational constant is denoted by `g` and the (possibly) variable bottom topography function ``b(x)``.
+Conservative variable water height ``h`` is measured from the bottom topography ``b``, therefore one
+also defines the total water height as ``H = h + b``.
+
+The additional quantity ``H_0`` is also available to store a reference value for the total water height that
+is useful to set initial conditions or test the "lake-at-rest" well-balancedness.
+
+Also, there are two thresholds which prevent numerical problems as well as instabilities. Both of them do not
```

Comment:
Just a reminder for us, we need to remember to update the docstrings in Trixi main once these thresholds move.

Reply:
Yes, good point! I started already to prepare the separation and removed them in my branch https://github.com/patrickersing/Trixi.jl/tree/TrixiSW-separation